src/GitVersionTfsTask/Extension/vss-extension.json

```diff
@@ -46,27 +45,15 @@
     },
     "screenshots": [
         {
-            "path": "img/builds.png"
+            "path": "builds.png"
         },
         {
-            "path": "img/build-task.png"
+            "path": "build-task.png"
         }
     ],
     "content": {
         "details": {
             "path": "overview.md"
         }
-    },
-    "contributions": [
```

Comment:
no this just filled automatically by our tooling, if you decided to keep [the tooling](https://www.npmjs.com/package/gl-vsts-tasks-build-scripts), they are no longer needed.